pcsx2/gui/Panels/VideoPanel.cpp

```diff
@@ -293,7 +293,7 @@ Panels::VideoPanel::VideoPanel( wxWindow* parent ) :
 
 	m_restore_defaults = new wxButton(right, wxID_DEFAULT, _("Restore Defaults"));
 
-	m_check_SynchronousGS->SetToolTip( pxEt( L"Enable this if you think MTGS thread sync is causing crashes or graphical errors.")
+	m_check_SynchronousGS->SetToolTip( pxEt( L"Enable this if you think MTGS thread sync is causing crashes or graphical errors. For debugging to see if GS is running at the correct speed.")
```

Comment:
> For debugging to see if GS is running at the correct speed

Not sure what the validity behind this statement is and considering that I wouldn't feel comfortable saying it's one of the reasons the setting exists.

Reply:
To be honest i've heard multiple things about this setting, to a legacy setting to putting on a separate thread. I've already deleted Debug tag on the description itself.